TimeTableTests/Categories/Projects/CollectionViewCell/ProjectCollectionViewCellModelTests.swift

```diff
@@ -45,7 +45,7 @@ class ProjectCollectionViewCellModelTests: XCTestCase {
     
     func testConfigureUpdateView() {
         //Arrange
-        let color = UIColor(string: "0c0c0c")
+        let color = UIColor(hexString: "#0c0c0c")
```

Comment:
Unless the previous assignments, it adds readability, so it's good, but I propose to use different initializer which takes UInt32 like:
`init(hex: UInt32, alpha: CGFloat = 1.0)`
and initialize like that:
`let color = UIColor(hex: 0xab0d7f)`

This is safer solution, because compiler will take care of validating it.
On the other side I know it only test, but it can be useful in the future.

Reply:
If you would like to get simple initializer, talk to me